neurostuff/example_config.py

```diff
@@ -5,45 +5,57 @@
 
 
 class Config(object):
-    SERVER_NAME = 'localhost'  # Set to external server name in production
 
-    MIGRATIONS_DIR = '/migrations/migrations'
-    SQLALCHEMY_TRACK_MODIFICATIONS = False
-    WTF_CSRF_ENABLED = False
+    def __init__(self):
+        self.SERVER_NAME = 'localhost'  # Set to external server name in production
 
-    POSTGRES_PASSWORD = os.environ.get('POSTGRES_PASSWORD', '')
-    DB_NAME = 'neurostuff'
-    SQLALCHEMY_DATABASE_URI = "postgres://postgres:" \
-        f"{POSTGRES_PASSWORD}@postgres:5432/{DB_NAME}"
-    PROPAGATE_EXCEPTIONS = True
+        self.MIGRATIONS_DIR = '/migrations/migrations'
+        self.SQLALCHEMY_TRACK_MODIFICATIONS = False
+        self.WTF_CSRF_ENABLED = False
 
-    GITHUB_CLIENT_ID = "github-id"
-    GITHUB_CLIENT_SECRET = "github-secret"
-    DANCE_SECRET_KEY = "temporary"
+        self.POSTGRES_PASSWORD = os.environ.get('POSTGRES_PASSWORD', '')
+        self.DB_NAME = 'neurostuff'
+        self.SQLALCHEMY_DATABASE_URI = "postgres://postgres:" \
+            f"{self.POSTGRES_PASSWORD}@postgres:5432/{self.DB_NAME}"
+        self.PROPAGATE_EXCEPTIONS = True
 
-    SECURITY_PASSWORD_HASH = 'pbkdf2_sha512'
-    SECURITY_PASSWORD_SALT = 'A_SECRET'
+        self.GITHUB_CLIENT_ID = "github-id"
+        self.GITHUB_CLIENT_SECRET = "github-secret"
+        self.DANCE_SECRET_KEY = "temporary"
+
+        self.SECURITY_PASSWORD_HASH = 'pbkdf2_sha512'
+        self.SECURITY_PASSWORD_SALT = 'A_SECRET'
 
 
 class ProductionConfig(Config):
-    ENV = 'production'
+    def __init__(self):
+        super().__init__()
+        self.ENV = 'production'
```

Comment:
```suggestion
        ENV = 'production'
```

As an example, I think this should work, since the `ENV` variable for this derivative class will be set before the `__init__` method is called. 

That way, no need to mess around with `super`.